searx/network/client.py

```diff
@@ -26,9 +26,6 @@
 logger = logger.getChild('searx.network.client')
 LOOP = None
 SSLCONTEXTS: Dict[Any, SSLContext] = {}
-TRANSPORT_KWARGS = {
-    'trust_env': False,
```

Comment:
Always true now = use the environment variables SSL_CERT_FILE and SSL_CERT_DIR when they exists.